### PR TITLE
refactor: created/moved module specific styles for test status choice group

### PR DESCRIPTION
**Pull request by waabid**
#### Description of changes

Just moved styles from general details view scss file to module specific scss file.

image

#### Pull request checklist

- [n/a] Addresses an existing issue: Fixes #0000
- [x] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [n/a] Ran precheckin (`yarn precheckin`)
- [x] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/components/test-status-choice-group.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    float: left;
+    line-height: 24px;
+
+    .radio-label {
+        font-size: 14px;
+        padding: 0 0 0 26px;
+        display: inline-block;
+    }
+
+    :global {
```
Comment (smoralesd) [inline src/DetailsView/components/test-status-choice-group.scss]: out of curiosity: what `:global` stand for?

Comment (waabid) [reply src/DetailsView/components/test-status-choice-group.scss]: This is to say that the following classes are not declared in this module. As a result, these are not modified by css-loader. This is beneficial in the scenarios we depend on certain class names being X or Y from components we don't control (Office Fabric).

Comment (smoralesd) [reply src/DetailsView/components/test-status-choice-group.scss]: I see... so you can't import them on your components neither, right?

Comment (waabid) [reply src/DetailsView/components/test-status-choice-group.scss]: Correct!

**Pull request merged**